samples/cpp/tutorial_code/objectDetection/objectDetection.cpp

```diff
@@ -30,7 +30,7 @@ int main( void )
     if( !eyes_cascade.load( eyes_cascade_name ) ){ printf("--(!)Error loading eyes cascade\n"); return -1; };
 
     //-- 2. Read the video stream
-    capture.open( -1 );
```

Comment:
Well in theory -1 should open up a box with the available camera's, but for me that has been broken since ever :D maybe we should adapt it everywhere. Lets wait for some core dev to get back on this.